framework/src/main/java/com/github/dbmdz/flusswerk/framework/engine/Worker.java

```diff
@@ -71,12 +73,21 @@ void executeProcessing(Message message) {
 
   public void process(Message message) {
     Collection<? extends Message> messagesToSend;
+    Exception skip = null;
```

Comment:
Shouldn't this type be `SkipProcessingException`?

Reply:
True, that's much clearer.